src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -225,6 +225,80 @@ public bool TryGetTelemetryId(out Guid telemetryId)
                 }
             }
 
+            public async IAsyncEnumerable<SuggestedActionSet> GetSuggestedActionsAsync(
+                ISuggestedActionCategorySet requestedActionCategories,
+                SnapshotSpan range,
+                IUIThreadOperationContext operationContext)
```

Comment:
this is the API we think we'll be implementing from the platform.  However, this is still in teh air as to what the final shape will be.